action.yaml

```diff
@@ -68,14 +68,15 @@ runs:
       shell: bash
       if: inputs.github-token != 'false'
       run: |
-        git remote set-url origin https://$DEPLOY_ACTOR:$DEPLOY_TOKEN@github.com/${OWNER_AND_REPOSITORY}
-        git config --global url.https://${DEPLOY_TOKEN}@github.com/.insteadOf git@github.com:
+        GITHUB_URL="https://${ACTOR}:${TOKEN}@github.com/"
+        git remote set-url origin "${GITHUB_URL}${OWNER_AND_REPOSITORY}"
+        git config --global url.${GITHUB_URL}.insteadOf git@github.com:
```

Comment:
`fatal: could not read Username for 'https://github.com/': No such device or address`

https://github.com/rentpath/tour_api/actions/runs/9423144796/job/25961081324#step:6:111